internal/impl/kafka/enterprise/redpanda_migrator_bundle_output.tmpl.yaml

```diff
@@ -87,29 +87,15 @@ mapping: |
         cases:
           - check: metadata("input_label") == "redpanda_migrator_input"
             output:
-              fallback:
-                - label: %s_redpanda_migrator_output
-                  redpanda_migrator: %s
-                  processors:
-                    - mapping: |
-                        meta input_label = deleted()
-                # TODO: Use a DLQ
-                - drop: {}
-                  processors:
-                    - log:
-                        message: |
-                          Dropping message: ${! content() } / ${! metadata() }
+              label: %s_redpanda_migrator_output
+              redpanda_migrator: %s
+              processors:
+                - mapping: |
+                    meta input_label = deleted()
           - check: metadata("input_label") == "redpanda_migrator_offsets_input"
             output:
-              fallback:
-                - label: %s_redpanda_migrator_offsets_output
-                  redpanda_migrator_offsets: %s
-                # TODO: Use a DLQ
-                - drop: {}
-                  processors:
-                    - log:
-                        message: |
-                          Dropping message: ${! content() } / ${! metadata() }
```

Comment:
We really shouldn't be dropping messages here... I left this as a TODO initially thinking that we might want to configure DLQs, but it's best to just block if we can't write data. If it's a transitive issue, it will resume when the broker becomes responsive again. If not, an operator will have to look into it. Restarting Migrator should be a safe operation (even if it potentially introduces duplicate records).